apis/cluster/v1beta1/membercluster_types.go

```diff
@@ -141,6 +141,23 @@ const (
 	// - "Unknown" means the member cluster has an unknown health status.
 	// NOTE: This condition type is currently unused.
 	ConditionTypeMemberClusterHealthy MemberClusterConditionType = "Healthy"
+
+	// ConditionTypeClusterPropertyProviderStarted indicates the startup condition of the configured
+	// cluster property provider (if any).
+	// Its condition status can be one of the following:
+	// - "True" means the cluster property provider has started.
+	// - "False" means the cluster property provider has failed to start.
+	// - "Unknown" means it is unknown whether the cluster property provider has started or not.
+	ConditionTypeClusterPropertyProviderStarted MemberClusterConditionType = "ClusterPropertyProviderStarted"
```

Comment:
At this moment there is no plan to support multiple property providers running at the same time, even though technically speaking it's obviously doable. I think it might be a little bit weird to use the `AllClusterPropertyProvidersStarted` reference now, but it could make sense if we decide to support so.